content/docs/refs-and-the-dom.md

```diff
@@ -31,30 +31,37 @@ Your first inclination may be to use refs to "make things happen" in your app. I
 
 ### Adding a Ref to a DOM Element
 
+>**Note:**
+>
+>The examples below have updated to use the new `React.createRef()` API, introduced in React 16.3.
+
+
 React supports a special attribute that you can attach to any component. The `ref` attribute takes a callback function, and the callback will be executed immediately after the component is mounted or unmounted.
 
-When the `ref` attribute is used on an HTML element, the `ref` callback receives the underlying DOM element as its argument. For example, this code uses the `ref` callback to store a reference to a DOM node:
+When the `ref` attribute is used on an HTML element, the `ref` created in the constructor with `React.createRef()` receives the underlying DOM element as its `value` property. For example, this code uses a `ref` to store a reference to a DOM node:
 
 ```javascript{8,9,19}
 class CustomTextInput extends React.Component {
   constructor(props) {
     super(props);
+    // create a ref to store the textInput DOM element
+    this.textInput = React.createRef();
     this.focusTextInput = this.focusTextInput.bind(this);
   }
 
   focusTextInput() {
     // Explicitly focus the text input using the raw DOM API
-    this.textInput.focus();
+    this.textInput.value.focus();
   }
 
   render() {
-    // Use the `ref` callback to store a reference to the text input DOM
-    // element in an instance field (for example, this.textInput).
+    // tell React that we want the associate the <input> ref
+    // to the `textInput` that we created in the constructor
```

Comment:
"associate _with_" rather than "associate _to_"?

Or maybe better yet:
> // Tell React to store a reference to the text input DOM
> // element in the `textInput` ref we created in the constructor.